app/Kconfig

```diff
@@ -388,11 +388,17 @@ if ZMK_BATTERY_REPORTING
 choice ZMK_BATTERY_REPORTING_FETCH_MODE
     prompt "Battery Reporting Fetch Mode"
 
-config ZMK_BATTERY_REPORTING_FETCH_MODE_STATE_OF_CHARGE
-    bool "State of charge"
-
 config ZMK_BATTERY_REPORTING_FETCH_MODE_LITHIUM_VOLTAGE
     bool "Lithium Voltage"
+    help
+        Estimate the battery level by reading the voltage of a Lithium battery.
+        zmk,battery chosen node should be a sensor driver that implements SENSOR_CHAN_VOLTAGE.
+
+config ZMK_BATTERY_REPORTING_FETCH_MODE_STATE_OF_CHARGE
```

Comment:
To avoid breaking existing out of tree boards that use a fuel gauge, I wonder if it might be reasonable to default this choice selection *if* our max17048 driver is enabled? Seems pragmatic until we can get the few out of tree boards to update their setup, then we remove that temporary work around?